CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- When a user checks permissions of a non-existing role or a non-existing resource, Conjur now audits a failure message.
```

Comment:
did @shulifink go over this? i think that we should change it to
```
Conjur now audits a failure message when a user checks permissions of a non-existing role or a non-existing resource
``

Reply:
Yeah, I consulted with her, and thats what she chose